eng/common/scripts/git-branch-push.ps1

```diff
@@ -61,8 +69,14 @@ if ($LASTEXITCODE -ne 0)
 }
 
 if (!$SkipCommit) {
-    Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -am `"$($CommitMsg)`""
-    git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" commit -am "$($CommitMsg)"
+    if ($AmendCommit) {
+        Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" commit -am `"$($CommitMsg)`""
```

Comment:
Is there a reason we needed to duplicate the full command in both of these cases? It seems like we could just add "--amend" in the case we want to append.